src/SIM/SIMbase.C

```diff
@@ -2544,9 +2543,8 @@ bool SIMbase::injectPatchSolution (Vector& sol, const Vector& vec,
       nndof != getNoFields(basis) && this->getNoFields(2) > 0) {
```

Comment:
og hvis ikke hvorfor jeg må sjekke for mixed gir mening umiddelbart: i normal ASM brukes ikke madof, men kun mlgn og nndof. følgelig funker det fint der, det er med mixed (når vi har flere nodetyper med forskjellig antall dofs) at komplikasjonen må til.